src/Modal.js

```diff
@@ -236,6 +207,8 @@ const styles = StyleSheet.create({
     right: 0,
     backgroundColor: "black",
     opacity: 0,
+    width: "100%",
+    height: "100%",
```

Comment:
I'm still testing it, but I think we can avoid setting these if we're already setting all the top, left, bottom, right elements to 0... what do you think?

Reply:
Yes, you are right 👍 I have updated PR.